apps/ui/src/views/Space/Turbo.vue

```diff
@@ -1,20 +1,36 @@
 <script setup lang="ts">
+import { TOKENS } from '@/composables/usePayment';
 import { _n } from '@/helpers/utils';
+import { metadataNetwork } from '@/networks';
+import { ChainId, Space } from '@/types';
 import ICInfinity from '~icons/c/infinity.svg';
 import ICPro from '~icons/c/pro.svg';
 import ICCheck from '~icons/heroicons-outline/check.vue';
```

Comment:
There is an issue here that block dev script to run, should remove .vue. Also actually we dont need to import the icon they are available like this <IS-check />